### PR TITLE
add stat function

### PR DESCRIPTION
**Pull request by fjakobs**
adds stat function

### CODE REVIEW

**Comment by fjakobs** [comment]
@creationix

File: README.markdown [hunk 1]
```diff
@@ -88,6 +88,28 @@ directory.  Entries contain:
...
 
+## vfs.stat(path, options, callback)
+
+Returns the file system attributes of a directory or a file and returns it
+using the same format as the `readdir` command.
+
+`options` can include:
+
+ - options.etag - the browser sent an If-None-Match header with etag
+ - options.head - the request was a HEAD request
```
Comment (creationix) [inline README.markdown]: I don't see code for this .head option anywhere in the code.  Also it doesn't make sense since the body isn't a stream currently.

Comment (fjakobs) [inline README.markdown]: True, head doesn't make sense. I'll remove it.

Comment (creationix) [inline README.markdown]: The thing is HEAD should be implemented for anything that's a GET request or we're in violation of the spec.  But if this isn't exposed as  GET request then there cannot be HEAD.

**Pull request merged**